implementations/micrometer-registry-signalfx/src/main/java/io/micrometer/signalfx/SignalfxDistributionSummary.java

```diff
@@ -54,7 +54,8 @@ final class SignalfxDistributionSummary extends AbstractDistributionSummary {
         super(id, clock, CumulativeHistogramConfigUtil.updateConfig(distributionStatisticConfig), scale, false);
         this.countTotal = new StepTuple2<>(clock, stepMillis, 0L, 0.0, count::sumThenReset, total::sumThenReset);
         max = new TimeWindowMax(clock, distributionStatisticConfig);
-        if (isDelta) {
+        if (!distributionStatisticConfig.isPublishingPercentiles()
+                && distributionStatisticConfig.isPublishingHistogram() && isDelta) {
```

Comment:
Is it possible that someone wants to publish both percentiles and histogram (for any reason)?
In that case, I would do 
```suggestion
        if (distributionStatisticConfig.isPublishingHistogram() && isDelta) {
```

Reply:
If that is a possibility, though the current AbstractTimer does not do that, I think your suggestion is good. will change the timer as well.

Reply:
Done.